orchestra/model/configuration.py

```diff
@@ -36,7 +37,7 @@ def follow_redirects(url, max=3):
     new_url = None
     with TemporaryDirectory() as temporary_directory:
         returncode = try_run_internal_subprocess(
-            ["git", "clone", url, temporary_directory],
+            ["git", "clone", "--depth", "1", url, temporary_directory],
```

Comment:
Why?

Reply:
It speeds up the cloning of the repository by not fetching the whole history which we don't need since we're just looking for the `REDIRECT` file in `HEAD`.
It also generates less network traffic, which is handy in case the initial repository has a long history or some binary files were inadvertently committed to it.

Reply:
It feels like a quite hypothetical issue, but OK.